consensus/istanbul/core/roundchange.go

```diff
@@ -101,7 +101,7 @@ func (c *core) handleRoundChange(msg *message, src istanbul.Validator) error {
 			c.sendRoundChange(roundView.Round)
 		}
 		return nil
-	} else if num == int(2*c.valSet.F()+1) && (c.waitingForRoundChange || cv.Round.Cmp(roundView.Round) < 0) {
+	} else if num == int(c.valSet.Size()-c.valSet.F()) && (c.waitingForRoundChange || cv.Round.Cmp(roundView.Round) < 0) {
```

Comment:
thinking a bit more about this threshold, or "quorum", value. the original `2f+1` is the minimum quorum size for byzantine tolerance, while the proposed `n-f` is the most stringent quorum size. We may want to consider a middle ground, using `ceil(2n/3)`.

| N | f | 2f+1 | ceil(2N/3) | N-f |
|---|---|---|---|---|
| 2 | 0 | 1 | 2 | 2 |
| 3 | 0 | 1 | 2 | 3 |
| 4 | 1 | 3 | 3 | 3 |
| 5 | 1 | 3 | 4 | 4 |
| 6 | 1 | 3 | 4 | 5 |